lib/seattleflu/id3c/cli/command/reportable_conditions.py

```diff
@@ -98,7 +98,11 @@ def webhook(suffix):
                         "Inferring site from manifest data.")
 
                 url = SLACK_WEBHOOK_REPORTING_GENERAL \
-                    if record.site not in childrens_sites and record.sheet != 'SCH' \
+                    if (record.site not in childrens_sites and
+                        record.sheet != 'SCH' and
+                        record.sample_origin != 'sch_retro' and
+                        record.swab_site != 'sch_ed' and
+                        record.swab_site != 'community_clinic') \
```

Comment:
Thanks for doing this!

There's so much negative logic here I think it might be preferable to flip the body of the if/else statements and check for positive matches.

e.g. `if (record.site in childrens_sites ...)` then use the Children's Slack webhook

Reply:
or we could pull the `!` to the front of the parentheses

Reply:
I have no preference, either way works. Flipping the if/else statement will probably make this easier to read.